frontend/src/api/Game.ts

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+import { axiosErrorHandler } from './Error';
+import { User } from './User';
+
+export type Room = {
+  roomId: string;
+  host: User;
+  users: [User],
+};
+
+export type StartGameParams = {
+  roomId: string,
+  initiator: User,
+};
+
+export const startGame = (params: StartGameParams):
+  Promise<void> => axios.post<void>('/api/v1/rooms/{currentRoomId}/start', params)
```

Comment:
This POST request fails - I think you need to use a template string (which means you'll also need take in another parameter currentRoomId as I don't think that exists here yet)

![Screenshot](https://user-images.githubusercontent.com/24768574/95283644-28636180-0811-11eb-827e-0a729e639103.png)

Reply:
As it turns out I forgot I was programming in Java and used `!=` to test string equality :)